### ISSUE TITLE
SSL problem

### ISSUE THREAD

**Issue by PabloMK7** [Rinnegatamante/lpp-3ds#44]
Trying to connect to a SSL server randomly causes "Failed creating SSL context". This error persists until you reboot your console. (Tried both with der certificate and without, same result).
Code: http://pastebin.com/jCuH4vzb

**Comment by Rinnegatamante** [comment]
Are you using latest nightly? @ksanislo pushed several patches for SSL connection.

**Comment by mariohackandglitch** [comment]
Bug confirmed in fc3e369

**Comment by mariohackandglitch** [comment]
I've debugged the problem, and I found out that sslccreatecontext returns: D8A0B801
Level: LEVEL_PERMANENT (Program error where retries are not possible.) 
Summary: SUMMARY_INVALID_STATE (State invalid. Might be initialised, or processing might be out of order.) 
Module: MODULE_NN_SSL 
Description: ???? (1)
I asked someone for help and told me to add SOC_AddGlobalSocket as stated [here](https://www.3dbrew.org/wiki/SSLC:CreateContext).
It didn't have any effects tho.
